backup/setup/roles.json

```diff
@@ -40,6 +40,7 @@
 		"HOMEWORK_CREATE",
 		"HOMEWORK_EDIT",
 		"HOMEWORK_VIEW",
+		"LERNSTORE_VIEW",
```

Comment:
the permission could be added to teachers instead, then users must not get LERNSTORE_HIDE applied

Reply:
this is about hiding the lernstore itself for students in certain locations. How would that work via teachers?

Reply:
This change adds some flexibility on permissions, while not breaking anything. Maybe we will have more cases when you have mutually exclusive permissions (you can be part of a group, or another group, but not both). In Nuxt we had already AND operator for permissions, in sidebar we had already excluded permissions option.